README.md

```diff
@@ -52,13 +52,19 @@ This is a work-in-progress implementation of a service broker for services provi
 `PASSWORD` - username password
 `DB_PREFIX` - a prefix that can be used to tag instances. Defaults to `compose-broker`
 `CLUSTER_NAME` - a name of your enterprise cluster if you've got one and want to use it. Defaults to hosted compose
-`ACCESS_TOKEN` - your API key for Compose.
+`COMPOSE_ACCESS_TOKEN` - your API key for Compose.
```

Comment:
I have second thoughts about this.  We would need to adjust `paas-cf` to support new variable: https://github.com/alphagov/paas-cf/blob/e070e920e7aa0f37300b4b97aa9f4709957e46da/concourse/pipelines/create-cloudfoundry.yml#L1509 ( might be in different places as well ) Also config still thinks it's `ACCESS_TOKEN` https://github.com/alphagov/paas-compose-broker/blob/add_elastic_search_tests/config/config.go#L65.

I'm 👍  with this change but we would have to do more PRs to make it happen.